tests/test_dataManager.py

```diff
@@ -63,13 +64,15 @@ def setUp(self):
         self.expected_header = {
             "FORMAT_V": 1,
             "INSTRUME": self.instrument,
-            "ORIGIN": self.origin,
+            "SERIAL": self.serial,
+            "CSCNAME": self.origin,
             "DETSIZE": self.n_pixels,
             "DATE-BEG": "1999-01-01T00:00:00.000",
             "DATE-END": "1999-01-01T00:00:02.500",
+            "DAYOBS": "19990101",
```

Comment:
DAYOBS is meant to be an integer. Also, maybe include OBSID and SEQNUM in this test code?

Reply:
Made the int() change to dataManager as well. Didn't add OBSID and SEQNUM in this test because they are generated in csc.py and it breaks the tests.